admin.php

```diff
@@ -90,6 +91,7 @@ function get_safe_path(string $path): string {
     </body>
 </html>
 <?php
+    fillContent('');
```

Comment:
initially fill the content with the home page. Not sure if this belongs here. Maybe it should rather be after the upload.